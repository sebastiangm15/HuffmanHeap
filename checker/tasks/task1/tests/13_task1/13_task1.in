12
5 U1
3 U2
7 V1
6 V2
10 W1
8 W2
15 X1
12 X2
9 Y1
4 Y2
13 Z1
2 Z2
