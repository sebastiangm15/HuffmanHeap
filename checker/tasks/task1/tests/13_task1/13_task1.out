94-Y1Y2U1W1Z2U2V2X2Z1V1W2X1 
39-Y1Y2U1W1Z2U2V2 55-X2Z1V1W2X1 
18-Y1Y2U1 21-W1Z2U2V2 25-X2Z1 30-V1W2X1 
9-Y1 9-Y2U1 10-W1 11-Z2U2V2 12-X2 13-Z1 15-V1W2 15-X1 
4-Y2 5-U1 5-Z2U2 6-V2 7-V1 8-W2 
2-Z2 3-U2 
